invenio_oaiserver/percolator.py

```diff
@@ -75,15 +81,18 @@ def _get_percolator_doc_type(index):
 }
 
 
-def _new_percolator(spec, search_pattern):
+def _new_percolator(spec, search_pattern, search_fields=None):
     """Create new percolator associated with the new set."""
     if spec and search_pattern:
-        query = query_string_parser(search_pattern=search_pattern).to_dict()
+        query = query_string_parser(
+            search_pattern=search_pattern,
+            search_fields=search_fields).to_dict()
         for index in current_search.mappings.keys():
             # Create the percolator doc_type in the existing index for >= ES5
             # TODO: Consider doing this only once in app initialization
             percolator_doc_type = _get_percolator_doc_type(index)
             _create_percolator_mapping(index, percolator_doc_type)
+            import ipdb; ipdb.set_trace()
```

Comment:
pytest :-)

Reply:
ah no it's a draft :) But thanks for the observation ;)